### PR TITLE
TILA-1691: Allow sorting units by name

### PR DESCRIPTION
**Pull request by saaste**
Adds `UnitsFiltersSet` and enables sorting `Unit` by name. I also added some tests to cover basic query cases.

### CODE REVIEW

**Review by nicobav — CHANGES_REQUESTED** [review]
Very nice :ok_hand: 
Only my vainglorius nitpicks about :nerd_face:

File: api/graphql/tests/test_units.py [hunk 1]
```diff
@@ -9,7 +10,7 @@
 from spaces.tests.factories import UnitFactory
 
 
-class UnitsUpdateTestCase(GraphQLTestCase):
+class UnitQueryTestCaseBase(GraphQLTestCase, snapshottest.TestCase):
     @classmethod
     def setUpTestData(cls):
         cls.unit = UnitFactory()
```
[the remark this answers is not in the document]
Comment (saaste) [reply api/graphql/tests/test_units.py]: Good idea! I added some 🍖

File: api/graphql/tests/test_units.py [hunk 2]
```diff
@@ -66,3 +69,79 @@ def test_normal_user_cannot_update_unit(self):
             "No permission to mutate"
         )
         assert_that(Unit.objects.get(pk=self.unit.pk).description).is_empty()
+
+    def test_getting_units(self):
```
[the remark this answers is not in the document]
Comment (saaste) [reply api/graphql/tests/test_units.py:82]: Now it is split into two ☺️

**Review by nicobav — APPROVED** [review]
Awesome :metal:

**Pull request merged**